src/dotnet/commands/dotnet-crossgen/CrossGenTarget.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Tools.CrossGen
+{
+    public class CrossGenTarget
+    {
+        public NuGetFramework Framework { get; private set; }
+        public string RID { get; private set; }
```

Comment:
Or "RuntimeIdentifier"

---
In reply to: [80278936](https://github.com/dotnet/cli/pull/4192#discussion_r80278936) [](ancestors = 80278936)